src/components/common/Modal/index.js

```diff
@@ -21,7 +21,7 @@ type Props = {|
 const Index = ({
   children,
   id,
-  modalSize = 'md',
+  modalSize = 'md', // TODO: make this actually change
```

Comment:
TODO?

Reply:
Yeah, it's part of the library that was around. I added the TODO to spare someone to try to change the modalSize by changing the parameter -- there is nothing implemented